pgcontents/crypto.py

```diff
@@ -0,0 +1,67 @@
+"""
+Interface definition for encryption/decryption plugins for
+PostgresContentsManager.
+
+Encryption backends should raise pgcontents.error.CorruptedFile if they
+encounter an input that they cannot decrypt.
+"""
+from .error import CorruptedFile
+
+
+class NoEncryption(object):
+    """
+    No-op encryption backend.
+
+    encrypt() and decrypt() simply return their inputs.
+
+    Methods
+    -------
+    encrypt : callable[bytes -> bytes]
+    decrypt : callable[bytes -> bytes]
+    """
+    def encrypt(self, b):
+        return b
+
+    def decrypt(self, b):
+        return b
+
+
+class FernetEncryption(object):
+    """
+    Notebook encryption using cryptography.fernet for symmetric-key encryption.
+
+    Parameters
+    ----------
+    fernet : cryptography.fernet.Fernet
```

Comment:
It looks like this implementation will work perfectly, unmodified, with a `MultiFernet` as well. `MultiFernet` enables no-downtime key rotation, which is a nice. Might be worth noting here.

(https://cryptography.io/en/latest/fernet/#cryptography.fernet.MultiFernet)